src/wayscript/utils.py

```diff
@@ -11,13 +12,36 @@ def get_process_execution_user_token():
     token = os.environ.get("WAYSCRIPT_EXECUTION_USER_TOKEN")
     return token
 
+def set_process_execution_user_token(value: str):
+    os.environ["WAYSCRIPT_EXECUTION_USER_TOKEN"] = value
+    return value
 
 def get_process_id():
     """Return uuid of current container execution"""
     process_id = os.environ["WS_PROCESS_ID"]
     return process_id
 
 
+def get_refresh_token():
+    refresh_token = os.environ["WAYSCRIPT_EXECUTION_USER_REFRESH_TOKEN"]
```

Comment:
Are we already setting this?

Reply:
I don't see it when calling `printenv`

Reply:
no, made it more gentle.

Reply:
I mean, I don't think the `WAYSCRIPT_EXECUTION_USER_REFRESH_TOKEN` is ever set from the server side?

Reply:
Oooh yes I see. I must have lost the change in a merge or rebase. Added back.